scaffold/convert/create-test.template.js

```diff
@@ -0,0 +1,30 @@
+require('should');
+
+const zapier = require('zapier-platform-core');
+
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+
+const utils = require('../utils');
+
+describe('Creates - <%= KEY %>', () => {
+  zapier.tools.env.inject();
+
+  it('should create an object', (done) => {
+    const bundle = {
+      authData: utils.getAuthData(),
```

Comment:
I'd try to create some method above (that would differ depending on the app's auth type) and call it here instead.

Reply:
Fixed all the `authData` issues in bf3c107.